tests/integration/cloudformation/resources/test_apigateway.py

```diff
@@ -99,14 +99,12 @@ def test_cfn_apigateway_aws_integration(
     assert mappings[0] == "(none)"
 
 
-@pytest.mark.skip_offline
 @pytest.mark.aws_validated
-def test_cfn_apigateway_swagger_import(deploy_cfn_template, apigateway_client):
+def test_cfn_apigateway_swagger_import(deploy_cfn_template, apigateway_client, echo_http_server):
     api_name = f"rest-api-{short_uid()}"
-    int_uri = "http://httpbin.org/post"
     deploy_cfn_template(
         template=TEST_TEMPLATE_1,
-        parameters={"ApiName": api_name, "IntegrationUri": int_uri},
+        parameters={"ApiName": api_name, "IntegrationUri": f"{echo_http_server}/post"},
```

Comment:
I like that much better that we're using the `http_server` fixture for our tests, it will make the tests much less flaky. However, we won't be able to test them anymore against AWS this way, or it would need a bit more setup. Should we leave some comment on how to AWS validate the test, maybe replacing manually the `int_uri` to the `httpbin` one?

Reply:
Done - added a `echo_http_server_post` fixture that can handle POST requests for both, local testing and AWS parity tests.